docs/source/pillars/computing_technology.md

```diff
@@ -70,6 +70,11 @@ _The tools used by researchers inside a TRE, such as programming languages, IDEs
   - Reproducibility of analyses improves auditability and accountability of how data has been used, as well as being best-practice in research.
     This may include version control software, and tools for developing and running data analysis pipelines.
   - Recommended
+* - Where possible, the TRE should automatically apply updates for user software.
```

Comment:
```suggestion
* - Where possible, the TRE should automatically apply security related updates for user software.
```
The downside of automatic updates is the software vendor makes a breaking change to an API, functionality, or UI.